spark-cassandra-connector/src/main/scala/com/datastax/spark/connector/types/TimestampFormatter.scala

```diff
@@ -2,7 +2,6 @@ package com.datastax.spark.connector.types
 
 import java.util.Date
 
-import org.apache.cassandra.serializers.TimestampSerializer
```

Comment:
@JeremiahDJordan The full changes required to remove our dependency on cassandra-clientutil :\